### PR TITLE
Physics expansion and Docs overhaul for version update

### PR DESCRIPTION
**Pull request by almilder**
Added:
- TS w gain
- new BG algs
- new background visualization
- new timing visualization
- interactive mode

Modified:
- Re-enabled series mode
- Visualization of fitting regions was improved
- Complete overhaul of docs (updated to reflect current code options, added more details)
- Rebuilt examples to work with current version

### CODE REVIEW

**Review by joglekara — CHANGES_REQUESTED** [review]
Thanks for getting this together — lots of good stuff in here, and the docs overhaul is going to make onboarding much easier. I went through the Python changes; the physics changes (Va per species, χ_EI sign, gain) all look intentional and trace cleanly to commits. A few things I'd like to ask you to fix before we merge:

**1. Loss function — duplicated EPWr term (blocker)**

In `tsadar/inverse/loss_function.py` around line 305, when the covar/else branch was added, the original `e_error += reduce_func(_error_)` line didn't get removed. So in the default (non-covar) path, the EPWr error is added twice:

```python
if self.cfg[\"optimizer\"][\"loss_method\"] == \"covar\":
    ...
    e_error += jnp.sum(...)/norm
else:
    e_error += reduce_func(_error_)
e_error += reduce_func(_error_)   # ← this line should be deleted
if self.cfg[\"data\"][\"fit_EPWb\"]:
    e_error *= 1.0 / 2.0
```

The fits probably still converge, but to a biased optimum since the `*= 0.5` only halves the doubled red contribution and leaves blue at its original weight. Just deleting that one line should fix it.

**2. `_1d_optax_loop_` will crash for any optimizer except `lbfgs`**

In `tsadar/inverse/loops.py` around line 99:

```python
opt.update(grad, opt_state, diff_params, value=..., grad=..., value_fn=loss_fn._loss_)
```

Those extra kwargs are lbfgs-only — `optax.adam`, `sgd`, etc. will raise `TypeError`. If the intent is \"this loop is lbfgs-only now,\" could you either gate those kwargs on the method name, or add an assert at the top so a stale `method: adam` config fails loudly? Right now it'd be a confusing crash mid-fit.

**3. Covariance loss path looks WIP**

In `calc_ei_error` and `calculate_covariance_matrix` there are leftover debug `print(norm)` / `print(x)` / `print(e_error)` statements that'll spam stdout for any user who selects `loss_method: covar`. Could you either strip those or, if covar isn't ready for general use yet, raise a clear \"not implemented\" error so it can't be selected accidentally? Happy to keep the implementation in the branch as you continue developing it — just want it walled off.

**Heads-up, not blocking:**

- The forward-pass tests had their tolerances loosened from `rtol=1e-4` to `atol≈1e-2` (`test_1d.py`, `test_angular_1v.py`). I see the comment that you investigated and the residuals are small — fine for now, but it does mean future form-factor regressions can hide in that band. Worth a follow-up issue to track down where the ~0.006 discrepancy comes from when you have time.
- \`Va\` moved from \`params.general.Va\` to per-ion-species. Any old configs floating around will \`KeyError\`. Consider noting in the release notes or version bump.
- A few commented-out code blocks in \`form_factor.py\` and \`loops.py\` (alternative χ calc, old optimizer init) — would be nice to remove on the next pass once you're sure you don't need them.

The rest looks good — happy to re-review once #1–3 are addressed.

**Review by joglekara — APPROVED** [review]
Fix on #1 looks good. Leaving #2 (optax kwargs) and #3 (covar prints) per discussion — both are walled off in practice. LGTM.
